env.source.sample

```diff
@@ -1,6 +1,12 @@
 ### Autodetect container subsystem
 export CONTAINER_SUBSYS=${CONTAINER_SUBSYS:-"$(which podman >/dev/null 2>&1 && echo podman || echo docker)"}
 
+### Set the container repo and tag we want to use.
+### Uncomment this to use the pre-built images Set it to localhost to use local images
```

Comment:
Can we say here what the default is if this is not set since we're providing a value that's not the default? - looks like we're using `localhost` as the default in the code - though I also have a comment about that as well in that file.

Reply:
Good call.